core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -153,6 +154,104 @@ function (string $value): string {
                         ->booleanNode('reject_large_uploads')
                             ->defaultValue(false)
                         ->end()
+                        ->arrayNode('sizes')
+                            ->useAttributeAsKey('name')
+                            ->validate()
+                                ->always(
+                                    static function (array $value): array {
+                                        static $reservedImageSizeNames = [
+                                            ResizeConfigurationInterface::MODE_BOX,
+                                            ResizeConfigurationInterface::MODE_PROPORTIONAL,
+                                            ResizeConfigurationInterface::MODE_CROP,
+                                            'left_top',
```

Comment:
@ausi I wonder if we should add a `ResizeConfigurationInterface::CROP_MODES` constant so we do not have to duplicate this code?